docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.4'
+
+services:
+  skoruba.identityserver4.admin:
+    image: ${DOCKER_REGISTRY-}skoruba-identityserver4-admin
```

Comment:
This ` ${DOCKER_REGISTRY-}` where is this variable set? When building on Linux this threw some errors.

Reply:
I am using Visual Studio 2017, I have not tested in VSCode.

Reply:
Could you provide couple of points or tutorial you followed to compose everything in VS?

Reply:
https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker?view=aspnetcore-2.2